.env.example

```
@@ -1,48 +1,131 @@
+
+# ============================================================================
+#   DATABASE
+# ============================================================================
+# Envio manages these for `pnpm run dev`. Set them for self-hosted deployments.
+
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+POSTGRES_DB=envio
+POSTGRES_USER=postgres
+# Use a strong value in production: compose falls back to "postgres" when unset.
+POSTGRES_PASSWORD=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_PORT key

(UnorderedKey)